var/spack/repos/builtin/packages/planck-likelihood/package.py

```diff
@@ -0,0 +1,133 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+import os
+import shutil
+
+
+class PlanckLikelihood(Package):
+    """2015 Cosmic Microwave Background (CMB) spectra and likelihood code"""
+
+    homepage = "https://wiki.cosmos.esa.int/planckpla2015/index.php/CMB_spectrum_%26_Likelihood_Code"
+    url      = "http://irsa.ipac.caltech.edu/data/Planck/release_2/software/COM_Likelihood_Code-v2.0.R2.00.tar.bz2"
+
+    version('2.00', '7a081679ff249dc4f94fb7177e16e818')
```

Comment:
Ah, it's because the parser is broken. In develop:
```
$ spack url parse http://irsa.ipac.caltech.edu/data/Planck/release_2/software/COM_Likelihood_Code-v2.0.R2.00.tar.bz2
==> Parsing URL: http://irsa.ipac.caltech.edu/data/Planck/release_2/software/COM_Likelihood_Code-v2.0.R2.00.tar.bz2

==> Matched version regex 17: r'((\\d+\\.)*\\d+)$'
==> Matched  name   regex  8: r'([^/]+)2.00'

==> Detected:
    http://irsa.ipac.caltech.edu/data/Planck/release_2/software/COM_Likelihood_Code-v2.0.R2.00.tar.bz2
                                                                --------------------------~~~~
    name:    com-likelihood-code-v2-0-r
    version: 2.00

==> Substituting version 9.9.9b:
    http://irsa.ipac.caltech.edu/data/Planck/release_2/software/COM_Likelihood_Code-v2.0.R9.9.9b.tar.bz2
                                                                --------------------------~~~~~~
```
With #2972:
```
$ spack url parse http://irsa.ipac.caltech.edu/data/Planck/release_2/software/COM_Likelihood_Code-v2.0.R2.00.tar.bz2
==> Parsing URL: http://irsa.ipac.caltech.edu/data/Planck/release_2/software/COM_Likelihood_Code-v2.0.R2.00.tar.bz2

==> Matched version regex 12: r'^[a-zA-Z\\d+_]+-v?(\\d[\\da-zA-Z.]*)$'
==> Matched  name   regex  7: r'^([A-Za-z\\d+\\._-]+)$'

==> Detected:
    http://irsa.ipac.caltech.edu/data/Planck/release_2/software/COM_Likelihood_Code-v2.0.R2.00.tar.bz2
                                                                -------------------  ~~~~~~~~~
    name:    COM_Likelihood_Code
    version: 2.0.R2.00

==> Substituting version 9.9.9b:
    http://irsa.ipac.caltech.edu/data/Planck/release_2/software/COM_Likelihood_Code-v9.9.9b.tar.bz2
                                                                -------------------  ~~~~~~
```
Since it is likely to change when #2972 is merged, you should add it anyways.